rewrite-java/src/main/java/org/openrewrite/java/dataflow/FindLocalFlowPaths.java

```diff
@@ -64,7 +64,7 @@ public Expression visitExpression(Expression expression, ExecutionContext ctx) {
 
     @Override
     public Expression visitExpression(Expression expression, ExecutionContext ctx) {
-        dataflow().findSinks(spec).ifPresent(flow -> {
+        Dataflow.startingAt(getCursor()).findSinks(spec).ifPresent(flow -> {
```

Comment:
@jkschneider if you really want me to, I will make this a constructor instead. I think this reads more cleanly, but I defer to your ultimate decision here.